docs/output.md

```diff
@@ -67,13 +67,10 @@ Most plots are included in the HTML report (see above), but are also included in
     - `raw.matrix.tsv`: RMA background corrected matrix (Affy)
     - `normalised.matrix.tsv`: RMA background corrected and normalised intensities matrix (Affy)
   - `differential/`: Directory containing tables of differential statistics reported by differential modules such as DESeq2
-    - `[contrast_name].[deseq2|limma].results.tsv`: Results of DESeq2 differential analyis (RNA-seq) OR Limma differential analysis (Affymetrix arrays, GEO studies, Maxquant proteomics studies)
-    - `[contrast_name].[deseq2|limma].results_filtered.tsv`: Results of DESeq2 differential analyis (RNA-seq) OR Limma differential analysis (Affymetrix arrays, GEO studies, Maxquant proteomics studies); filtered for differentially abundant entries
```

Comment:
This change deletes quite a few changes that were done on the dev branch but not yet merged to master (the results_filtered files should be documented and Maxquant should be mentioned in both lines); please ensure that your branch is up-to-date with dev